web/src/components/Breadcrumbs.tsx

```diff
@@ -55,6 +55,8 @@ type PropsType = {
 const Breadcrumbs = ({ direction, ancestorBreadcrumbs, currentBreadcrumb }: PropsType): ReactElement => {
   // The current page should not be listed in the UI, but should be within the JsonLd.
   const jsonLdBreadcrumbs = [...ancestorBreadcrumbs, currentBreadcrumb]
+  // Min amount of chars when the last breadcrumb item should shrink
```

Comment:
I think `Min text length after which the last breadcrumb item should shrink` is more understandable.